.travisci/codecov.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+mvn jacoco:report
+"$HOME/.script/codecov.bash" -F"unit$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION"
+mvn jacoco:report-integration
+"$HOME/.script/codecov.bash" -F"compiler$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION"
+
+(cd target/it && 
+	for name in * ; do 
+		(cd "$name" && 
+			mvn jacoco:report && 
+				"$HOME/.script/codecov.bash" -s . -F"unit$(echo $name | sed 's/[+-]//g')$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION"  || echo 'NOT POSSIBLE');
+		(cd "$name" && 
+			mvn jacoco:report-integration && 
+				"$HOME/.script/codecov.bash" -s . -F"compiler$(echo $name | sed 's/[+-]//g')$TRAVIS_OS_NAME$TRAVIS_JDK_VERSION" || echo 'NOT POSSIBLE');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/powerunit/powerunit-extensions-matchers/pullRequest?prid=5559528)